pkg/crc/cluster/cluster.go

```diff
@@ -104,6 +105,29 @@ func GetRAMUsage(sshRunner *ssh.Runner) (int64, int64, error) {
 	return ramSize, ramUsage, nil
 }
 
+func GetCPUUsage(sshRunner *ssh.Runner) ([]int64, error) {
+	cmd := "top -n1 -1 -b -w 79  | grep %Cpu | awk '{print $3}'"
+	out, _, err := sshRunner.Run(cmd)
+
+	if err != nil {
+		return nil, err
+	}
+
+	cpuUsage := strings.Split(strings.TrimSpace(out), "\n")
+	result := make([]int64, len(cpuUsage))
+	for i, s := range cpuUsage {
+		c, err := strconv.ParseFloat(s, 64)
+		if err != nil {
+			return nil, err
+		}
+
+		result[i] = int64(math.Round(c))
```

Comment:
similarly to one of my previous suggestions ,you could use `result = append(result, int64(...))` which removes the need for allocating the correct amount of memory and using the right index.